connector/queue/model.py

```diff
@@ -224,6 +227,16 @@ def autovacuum(self):
         return True
 
 
+class QueueJobRecordRelation(models.BaseModel):
```

Comment:
Weren't you going to try without the 'id' column? I believe BaseModel still has that.

Reply:
I'm not sure how to even achieve this. Also, I created an extension to the `queue.job` view in order to display the records and the models.

Reply:
Yes alright then.